src/components/Canvas/Canvas.vue

```diff
@@ -63,18 +56,17 @@
 
 <script>
 import defu from 'defu'
-import Intersect from 'vue-intersect'
```

Comment:
This wasn't compatible with vue 3 so I just removed it. It seems like it's for optimization reasons, but if we still want that we may need to use [this function from vue core](https://vueuse.org/core/useIntersectionObserver/).